config/application.yml.example

```diff
@@ -19,6 +19,12 @@ mailer_domain_name: 'http://localhost:3000'
 # https://github.com/dropbox/zxcvbn#usage
 min_password_score: '3'
 
+# How often to enqueue simple jobs to make sure the backgound queues are running
+queue_health_check_frequency_seconds: '120'
+
+# If a queue does not have a healthy job after this many seconds, report it as unhealthy
+queue_health_check_dead_interval_seconds: '240'
+
```

Comment:
If we always want these values to be the same for all environments, then we only need to define them once, above the environment keys. Otherwise, we would need them under every environment section, but not at the top.